README.md

```diff
@@ -75,3 +81,11 @@ Topic:topic2    PartitionCount:5        ReplicationFactor:1     Configs:retentio
 ```
 
 Any changes to the `test-topics.yml` file will be applied to the existing topics at each subsequent run.
+
+##### Using the Docker image
+
+Assuming you know the <zookeeper_address> and have placed your config file named test-topics.yml inside the <config_dir_on_host> directory on the Docker host, an example of how to run the image is:
```

Comment:
Missing backticks around `<zookeeeper_address>`, `test-topics.yml`, `<config_dir_on_host>` in my opinion